www/install/php/Update-20.04.0-beta.1.php

```diff
@@ -17,3 +17,16 @@
  * For more information : contact@centreon.com
  *
  */
+include_once __DIR__ . "/../../class/centreonLog.class.php";
+$centreonLog = new CentreonLog();
+
+try {
+    $pearDB->query(
+        "UPDATE `contact` SET `contact_autologin_key` = NULL WHERE `contact_autologin_key` =''"
+    );
+} catch (\PDOException $e) {
+    $centreonLog->insertLog(
```

Comment:
I'm not sure this is a best way to silently fail during upgrade.
But we will have a more general discussion later about it